pom.xml

```diff
@@ -1306,37 +1238,6 @@
                 <artifactId>jsoup</artifactId>
                 <version>${version.jsoup}</version>
             </dependency>
-
-            <!-- Solve dependency convergence issues through exclusions for transitive-only dependencies -->
-            <dependency>
-                <groupId>org.glassfish.jaxb</groupId>
-                <artifactId>jaxb-core</artifactId>
-                <!-- DO NOT MANAGE the version of this dependency. -->
-                <exclusions>
-                    <!-- Also imported by Hibernate ORM -->
-                    <exclusion>
-                        <groupId>jakarta.xml.bind</groupId>
-                        <artifactId>jakarta.xml.bind-api</artifactId>
-                    </exclusion>
-                    <!-- Also imported by jakarta.xml.bind-api -->
-                    <exclusion>
-                        <groupId>com.sun.activation</groupId>
-                        <artifactId>jakarta.activation</artifactId>
-                    </exclusion>
-                </exclusions>
-            </dependency>
-            <dependency>
-                <groupId>org.glassfish.jaxb</groupId>
-                <artifactId>jaxb-runtime</artifactId>
-                <!-- DO NOT MANAGE the version of this dependency. -->
-                <exclusions>
-                    <!-- Also imported by jakarta.xml.bind-api -->
-                    <exclusion>
-                        <groupId>com.sun.activation</groupId>
-                        <artifactId>jakarta.activation</artifactId>
-                    </exclusion>
-                </exclusions>
-            </dependency>
```

Comment:
Ah, see. Here. That's where we should solve dependency convergence issues related to jakarta.activation.

Reply:
Yes, it was there in the root pom. But for some reason, maven was complaining about missing versions on these dependencies, and since there was a comment about not adding a version, I've tried to move it to the ORM mapper module since that's where we'd be adding ORM. It may be that the reason maven complained was that one of these got listed as an explicit dependency in some module (I think it was the property collector one...) I'll try moving it back to the root pom and will let you know how it goes 😃

Reply:
Ok, thanks.

> I've tried to move it to the ORM mapper module since that's where we'd be adding ORM

FYI there are several other places where we add the ORM dependency independently. Mainly utils modules, I believe, but still, if the dependency management option doesn't work out, you might want to be careful about those.